go_utils.py

```diff
@@ -49,13 +55,11 @@ def make_go(instr_dict: InstrDict):
         instr_str += f"""  case A{i.upper().replace("_","")}:
     return &inst{{ {hex(opcode)}, {hex(funct3)}, {hex(rs1)}, {hex(rs2)}, {signed(csr,12)}, {hex(funct7)} }}
 """
+    for num, name in csrs:
```

Comment:
The entries in the source code are ordered by CSR number, rather than CSR name.  See

https://go-review.googlesource.com/c/go/+/630518/2/src/cmd/internal/obj/riscv/inst.go

Reply:
But I'll double check that the order is guaranteed.

Reply:
The ordering of the CSRs in our Go map is determined by the ordering of the CSRs in [csrs.csv](https://github.com/riscv/riscv-opcodes/blob/master/csrs.csv).

Reply:
I've updated the patch to sort the CSRs by CSR number.  The resulting inst.go file can be seen [here](https://go-review.googlesource.com/c/go/+/630518/3/src/cmd/internal/obj/riscv/inst.go)